app/src/main/java/org/wildaid/ofish/ui/home/HomeActivity.kt

```diff
@@ -78,7 +87,7 @@ class HomeActivity : AppCompatActivity(R.layout.activity_home) {
             getString(android.R.string.cancel)
         ).bundle()
 
-        navigation.navigate(R.id.action_homeFragment_to_ask_change_duty_dialog, dialogBundle)
+        navigation.navigate(R.id.ask_change_duty_dialog, dialogBundle)
```

Comment:
Is this renaming needed?

Reply:
Yes. After updating a libraries it started crashing if we use different source point -  Fragment tries to use Activity